opus/application/apps/help/templates/help/gettingstarted.html

```diff
@@ -179,6 +179,17 @@ <h3>The Cart</h3>
     of observations, from the <b>Browse Results</b> tab. The number of
     observations in the cart will be displayed next to the <b>Cart</b> tab.</p>
 
+<p>Editing directly on the <b>Cart</b> tab will place the edited observations
```

Comment:
(Look at this comment in preview mode to get the HTML markup in plain text)

How about this, since I don't know what "editing" means.

"While on the <b>Cart</b> tab, you can move observations to the recycle bin rather than removing them from the cart entirely. These observations will be marked with <recycle bin icon>. You can also restore observations from the recycle bin to the cart. Observations may be move to the recycle bin or restored to the cart as individual observations or as ranges of observations. In addition, all observations in the recycle bin can be moved to the cart at once by clicking on <b>Restore Recycle Bin to Cart</b>. All observations can be removed from the recycle bin by clicking on <b>Empty Recycle Bin</b>.

Observations in the recycle bin are not counted as part of the total result count, nor will they be included in the downloaded data. The total number of observations currently in the recycle bin will be displayed next to the <b>Restore Recycle Bin to Cart" option.